dockerfile/wildfly/microprofile/embed-server.cli

```diff
@@ -8,21 +8,42 @@ embed-server --std-out=echo
 run-batch --file=${TMP_DIR}/embed-server/configure-managed-executor.cli
 run-batch --file=${TMP_DIR}/embed-server/configure-undertow.cli
 
-#####Microprofile configs#####
-run-batch --file=${TMP_DIR}/embed-server/configure-opentracing.cli
+#####Microprofile telemetry#####
+# add telemetry with exception handling
+try
+/extension=org.wildfly.extension.opentelemetry:add
+/extension=org.wildfly.extension.microprofile.telemetry:add
+/subsystem=opentelemetry:add
+/subsystem=microprofile-telemetry:add
+catch
```

Comment:
ezt nem kellene külön bontani? így ha az opentelemetry már aktiválva van de a microprofile.telemetry még nincs akkor az ki fog maradni

Reply:
A WF admin guide szerint a ketto egyutt mozog, szerintem idovel fogjak modositani mert ellentmondas van a subsystemben. Ha valtozas lessz benne ugy is modositani fog kelleni